.circleci/organise_doc.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const fs = require('fs-extra');
+const args = require('minimist')(process.argv.slice(2));
+
+// get SDK name and version from package files
+// they are used by JSDoc to generate the output directories
+const PACKAGE_JSON = require('../package.json');
```

Comment:
[Standardjs](https://standardjs.com/) style is to use `camelCase` instead of `CAPITALIZED_UNDERSCORE_CASE`, except perhaps when defining "universal" constants that are defined from literals (e.g. `const HOURS_PER_DAY = 24;`)

Not really worth changing this unless you're changing other things, though.

Reply:
yes java dev habits... sorry